docs/src/whatsnew/3.4.rst

```diff
@@ -7,7 +7,7 @@ This document explains the changes made to Iris for this release
 (:doc:`View all changes <index>`.)
 
 
-.. dropdown:: :opticon:`report` |iris_version| Release Highlights
+.. dropdown:: :opticon:`report` v3.4 Release Highlights
```

Comment:
Should this instead be 
```suggestion
.. dropdown:: :opticon:`report` v3.4.0 Release Highlights
```

The other whats news include the patch number, see [3.3](https://github.com/SciTools/iris/blame/main/docs/src/whatsnew/3.3.rst#L10), [3.2](https://github.com/SciTools/iris/blame/main/docs/src/whatsnew/3.2.rst#L10) and [3.1](https://github.com/SciTools/iris/blame/main/docs/src/whatsnew/3.1.rst#L10)

Reply:
I had done this deliberately, but while trying to argue my corner in this comment I argued myself out of it 😄